pages/team-members.tsx

```diff
@@ -7,31 +7,45 @@ import TeamMemberForm from "../src/components/team-members/TeamMemberForm";
 import * as Styled from "../page-styling/PageContainer.styled";
 
 import { getTeamMembers } from "../src/services/getTeamMembers";
-import { StoreContext } from "../src/stores/StoreProvider";
+import { useStore } from "../src/stores/ZustandStore";
+import SearchBar from "../src/components/search-bar/SearchBar";
 
 function TeamMembersPage() {
   const [isOpen, setIsOpen] = useState<boolean>(true);
+  const [isVisible] = useState(true);
```

Comment:
Is this state effectively used anywhere? I wonder because there is no `setState`

Reply:
I use it to hide the filter option on the team members page

Reply:
If the value is never changed, it can probably be a regular variable instead of a state